docs/app/Examples/elements/Header/Content/HeaderImageExample.js

```diff
@@ -0,0 +1,16 @@
+import faker from 'faker';
+import React, {Component} from 'react';
+import {Header, Image} from 'stardust';
+
+export default class HeaderImageExample extends Component {
+  render() {
+    const photo = <Image className='circular' src={faker.image.city()} />;
+    return (
+      <Header.H2 image={photo}>
+        <div className='content'>
```

Comment:
Removing the `content` div breaks the formatting of the example. (See the screenshot) I'll look at how we're handling passing in `content` in the component itself
<img width="414" alt="screen shot 2015-11-19 at 9 03 22 am" src="https://cloud.githubusercontent.com/assets/6728034/11274073/8223dcc0-8e9c-11e5-9212-88f42d2e6d8c.png">